src/js/utils/DSLUtil.js

```diff
@@ -194,6 +194,19 @@ const DSLUtil = {
           memo[prop] = matchingNodes
             .map((ast) => ast.filterParams.text)
             .join(' ');
+
+          // Also append whatever whitespace remains at the end of the raw value
+          //
+          // NOTE: This is a trick to allow dynamic updates of a React component
+          //       since white spaces at the end of the expression are not part
+          //       of a regular DSL expression and are trimmed.
+          //
+          const whitespaceRegex = /\s+$/;
+          const tailingWaitspace = whitespaceRegex.exec(expression.value);
```

Comment:
![Wait for it](http://24.media.tumblr.com/tumblr_m3x648wxbj1ru99qvo1_250.png)
.. Space!